examples/falling-balls-mpp/build.gradle.kts

```diff
@@ -74,8 +62,6 @@ kotlin {
                     "-linker-option", "-framework", "-linker-option", "CoreText",
                     "-linker-option", "-framework", "-linker-option", "CoreGraphics"
                 )
-                // TODO: the current compose binary surprises LLVM, so disable checks for now.
-                freeCompilerArgs += "-Xdisable-phases=VerifyBitcode"
```

Comment:
Is it indeed no longer needed?

Reply:
It is already set https://github.com/JetBrains/compose-jb/pull/1682/files/242d2bb9d2746421e15a5e8a47385417aa6a32a5#diff-7367572e12d1f72a6493a4ded4868e01806a56006fe6deb1742634d4b1ed53dcR164